RNPurchases.podspec

```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |spec|
   spec.exclude_files = "ios/Purchases.framework"
 
   spec.dependency   "React"
-  spec.dependency   "PurchasesHybridCommon", "1.4.2"
+  spec.dependency   "PurchasesHybridCommon", "1.4.3"
```

Comment:
have you tried installing this as a pod? I think it would lead to some issues since it doesn't declare `static_framework = true`

Reply:
I actually haven't tried. My bad. Let me see

Reply:
hmmm it works. I tried running it and the app runs just fine. Should I add it just in case?

Reply:
Remember to delete the `podfile.lock` file before doing a `pod install`, otherwise it may no-op. 
I just tried it on an empty project, adding to the podfile: 

```ruby
pod 'RNPurchases', path: '~/revenuecat/react-native-purchases/'
```
and I got the error: 
```bash
[!] The 'Pods-cocoapodstest' target has transitive dependencies that include statically linked binaries: (PurchasesHybridCommon, Purchases, and PurchasesCoreSwift)
```

Reply:
![Screen Shot 2020-09-30 at 8 33 36 AM](https://user-images.githubusercontent.com/664544/94706965-b910dd80-02f7-11eb-92db-43d749630bea.png)

That's what I did, and still no error